### PR TITLE
f_op_actor_mng split & started decomp

### PR DESCRIPTION
**Pull request by lepelog**

### CODE REVIEW

File: include/JSystem/JMath/JMath.h [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+struct sinCosTableEntry {
+    f32 sin;
+    f32 cos;
+};
+
+extern sinCosTableEntry lbl_80439A20[0x2000];  // JMath::JMath::sincosTable_
+
+inline f32 cM_scos(s16 s) {
+    return lbl_80439A20[(s & 0xFFFF) >> 3].cos;
```
Comment (notyourav) [inline include/JSystem/JMath/JMath.h:14]:
```suggestion
    return lbl_80439A20[s >> 3].cos;
```
is this needed?

Comment (lepelog) [reply include/JSystem/JMath/JMath.h:14]: That change doesn't match, however `return lbl_80439A20[static_cast<u16>(s) >> 3].cos;` also matches, would that be preferred?

Comment (notyourav) [reply include/JSystem/JMath/JMath.h:14]: Hmm.. I would say yes, given the context (no negative array access). They may have had one here.

**Review by notyourav — APPROVED** [review]

File: include/JSystem/JMath/JMath.h (hunk 1 above)
Comment (lepelog) [reply include/JSystem/JMath/JMath.h:14]: Looking at debug, they had way more layers of inlined (template) functions here

**Pull request merged**